pkg/fakerp/config.go

```diff
@@ -61,43 +61,47 @@ func GetPluginTemplate() (*pluginapi.Config, error) {
 	template.Certificates.GenevaLogging.Key = logKey
 	template.Certificates.GenevaMetrics.Cert = metCert
 	template.Certificates.GenevaMetrics.Key = metKey
-	template.Images.GenevaImagePullSecret = pullSecret
-	template.Images.ImagePullSecret = imagePullSecret
+	template.GenevaImagePullSecret = pullSecret
+	template.ImagePullSecret = imagePullSecret
```

Comment:
This is sole image config. This is why it was under `Images`. Any reasoning to move/change api?

Reply:
yes - I am hoping the pull secrets are the same regardless of the cluster version